newswires/client/src/SideNav.tsx

```diff
@@ -134,7 +110,7 @@ export const SideNav = () => {
 					activeSuppliers.length === recognisedSuppliers.length,
 				onClick: () => handleEnterQuery({ ...config.query, supplier: [] }),
 				onTickerClick: () => {
-					openTicker('supplier', config);
+					openTicker({ ...config.query });
```

Comment:
To open all suppliers instead of only those currently selected:
```suggestion
					openTicker({...config.query, supplier: []});
```